.env.template

```diff
@@ -4,13 +4,17 @@ EIA_API_KEY=
 # Register at https://apiportal.pjm.com/ for an API key
 PJM_API_KEY=
 
+# NOTE: The ERCOT API is geo-restricted.
```

Comment:
If you want to run this, you'll need to use a VPN and set your location to Texas.